.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .DS_Store
 dist/loader.*
 dist/*.gz
+demo
```

Comment:
What's in `demo`?

Reply:
I think that's mistaken from a previous experiment I did with a different package.  This change in particular can be reverted, I think